web/common/src/components/Typography/Headline.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import { getHeadlineTextSize } from './help'
+import type { HeadlineLevel } from '@/types'
+import { cn } from '@/utils'
+
+export function Headline({
```

Comment:
This feels similar to me, this is just an h-tag which shouldn't be optional + two classes. Why do we need a component for that?